pkg/services/ppmshipment/ppm_shipment_updater.go

```diff
@@ -49,6 +51,19 @@ func (f *ppmShipmentUpdater) updatePPMShipment(appCtx appcontext.AppContext, ppm
 		return nil, err
 	}
 
+	newPPMEstimator := NewEstimatePPM()
+	estimator, err := newPPMEstimator.EstimateIncentiveWithDefaultChecks(appCtx, *oldPPMShipment, updatedPPMShipment)
+	// HOW DO YOU CALL THE ESTIMATOR?
+	verrs, _ := appCtx.DB().ValidateAndUpdate(estimator)
+	// Add check for if the estimator encounters an err, then do not proceed with update
+	if err != nil {
+		return nil, err
+	}
+
+	if verrs != nil && verrs.HasAny() {
+		return apperror.NewNotImplementedError("Cannot update the Estimated Incentive."), nil
```

Comment:
Regarding the first server test error:
<img width="1505" alt="Screen Shot 2022-04-11 at 1 22 00 PM" src="https://user-images.githubusercontent.com/67110378/162795281-70e2fbf4-bd65-42e6-a13d-18b7c6e1ecd1.png">

The error is coming up because the function expects the return statement to be `(*models.PPMShipment, error)` and currently it's `(error, *models.PPMShipment)`
```suggestion
		return nil, apperror.NewNotImplementedError("Cannot update the Estimated Incentive.")
```

Reply:
Hi there! Thanks for the review. I fixed the change right around when you wrote this, but my PR should be up to date now with all of the most recent changes.